cmd/a2o-migrate/test/Dockerfile

```diff
@@ -0,0 +1,4 @@
+#FROM balena/armv7hf-supervisor:v9.0.1
+FROM busybox
```

Comment:
Does it matter what base image is used for the test?

Reply:
not really I just wanted a smaller image that gets pulled for the test

Reply:
So should we remove the commented line?